src/pal/src/misc/stackstring.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+#include "pal/malloc.hpp"
+
+template <UINT32 MEMSIZE>
+class StackString
```

Comment:
This is really a buffer class - there is nothing "string" about it. Should it be called StackBuffer instead?